testsuite/integration/microprofile/pom.xml

```diff
@@ -277,6 +316,10 @@
                                                 <layer>cloud-server</layer>
                                                 <layer>microprofile-platform</layer>
                                                 <layer>ejb</layer>
+                                                <!-- TODO - keep the reactive ones here or put in own execution? -->
```

Comment:
Good question. 

I have no answer right now, but I suspect what you have here will survive due to inertia.

Reply:
Yeah, TBH I didn't go down the route I am indirectly suggesting since it is already quite complicated with all these profiles :)